www/js/modules/main/directives/histogram/histogramModel.js

```diff
@@ -0,0 +1,48 @@
+export default class HistogramModel {
+
+    constructor() {
+        // Root element Histogram is going to attach itself to DOM
+        this.rootElement            = null;
+        // Histogram Properties
+        this.svgHeight              = null;
+        this.svgWidth               = null;
+        this.barContainerHeight     = null;
+        this.barContainerWidth      = null;
+        this.margin                 = null; 
+        this.padding                = null; 
+        this.ticks                  = null; 
+        this.domainMax              = null;
+        this.barHeight              = null;
+
+        // D3 Scales
+        this.scaleBand              = null;
+    }
+
+    // Root Element
+    getRootElement()                       { return this.rootElement;                  }
+    setRootElement(element)                { this.rootElement = element;               }
```

Comment:
Both getRootElement and setRootElement work correctly here.  However, the lines below with the "set svgHeight" I get the RangeEorror on.